### DISCUSSION TITLE
Running Docker using GUI container management

### DISCUSSION THREAD

**Discussion by SubBass100** [aaronwmorris/indi-allsky#948] [Q&A]
Hey there! I'm newish to docker but have a general understanding of it. I've deployed container images like pihole and heimdall using it and everything was pretty straight forward.... Grab the docker-compose.yaml file, customize some settings and persistent storage then build. However, I'm having some trouble doing that with indi-allsky. I saw that there is a .env file to fill out which I did but it's not building right. Is there something special about this container than others to why it might not be working. I've ran indi-allsky on a pi and in a vm but think it would be better suited running as a docker image on my synology using container manager (my camera is an IP camera). Thanks in advance!

**Comment by aaronwmorris** [comment]
The only particularly special thing is the script to generate the .env file and related certificates.  If you did not use it, I would suggest deleting the .env file and regenerating it.  You may have to delete your docker volumes, as well.

The only other possible issue is you should make sure you are using docker compose v2 and not v1.  Usually, starting compose with `docker compose` (no dash) and not docker-compose is good enough.
